documentation/modules/ROOT/pages/hcp-tech-breakdown.adoc

```diff
@@ -29,45 +31,53 @@ This operator is versioned in lockstep with a specific OCP version and is decoup
 The main responsibilities of this operator are:
 
 * Provisioning all the infrastructure required to host a control plane (it can be creating it or adopting existing one). The infrastructure can refer to management cluster resources, external cloud provider resources, etc.
-* Deploying an OCP control plane configured to run in the context of the provisioned infrastructure.
-* Implementing any versioned behavior necessary to rollout the new version of a control plane.
+(what a control plane needs as infrastructure is fixed, no? if yes can we list one by one?)
+* Deploying an OCP control plane configured to run in the context (what is context?namespaced?) of the provisioned infrastructure.
+* Implementing any versioned behavior necessary (any example) to rollout the new version of a control plane.
```

Comment:
This would be an implementation detail. Like behavior that needs to be coded for going from 4.X to 4.Y